tasks/main.yml

```diff
@@ -1,10 +1,31 @@
 ---
 - include: Debian.yml
-  when: ansible_os_family == "Debian"
+  static: no
+  when: ansible_os_family == 'Debian'
 
 - include: RedHat.yml
+  static: no
   when: ansible_os_family == "RedHat"
 
+# Not a big fan of this but the default package explicitly prevents defining the hosts
+# option in the daemon config by adding a useless -H option on the dockerd start command,
+# this service file removes it
+- name: Update systemd definition to allow for custom listen options
+  template: src=docker.service.j2 dest=/lib/systemd/system/docker.service
```

Comment:
Shouldn't it use drop-in config files instead? (`/etc/systemd/system/docker.service.d/`)

It would be:
```
[Service]
EnvironmentFile=-/etc/default/docker
ExecStart=
ExecStart=/usr/bin/dockerd $DOCKER_OPTS
```